airflow/config_templates/default_airflow.cfg

```diff
@@ -1073,10 +1073,19 @@ worker_precheck = False
 # ETA you're planning to use.
 # visibility_timeout is only supported for Redis and SQS celery brokers.
 # See:
-# http://docs.celeryproject.org/en/master/userguide/configuration.html#std:setting-broker_transport_options
+# https://docs.celeryq.dev/en/stable/getting-started/backends-and-brokers/redis.html#visibility-timeout
 # Example: visibility_timeout = 21600
 # visibility_timeout =
 
+# The sentinel_kwargs parameter allows passing additional options to the Sentinel client.
+# In a typical scenario where Redis Sentinel is used as the broker and Redis servers are
```

Comment:
You should add them description in config.yml -> this is the source of truth for this defailt config but also it is used to generate documentation automatically, so the comment you added here, should be added in config.yml and then pre-commit should automatically add it above and you should not need to add it manually here.

Reply:
Actually, I don't add it manually. When I make modifications in config.yml and do a git commit, the pre-commit hook prevents my merge request and automatically generates this content in my git stage. Only after that, I can successfully commit without any pre-commit issues.

Reply:
The pre-commit prevents you doing it and modifes it and ask you to commit the changes it has done when it add. 
You need to:

1) add changes to ,yml
2) commit and let pre-commit add it to the config
3) add the generated config changes  it to the staging
4) commit again

Reply:
Ah ... stupid me. yes you re right! I missed that it is example generated from the other part. Sorry for the confusion.